src/components/product/product-name-creator.test.js

```diff
@@ -0,0 +1,59 @@
+import nameCreatorFactory, { stringifyStack } from './product-name-creator'
+
+describe('stringifyStack', () => {
+  it('verifies create string with product type', () => {
+    const string = stringifyStack([{ name: 'type', value: 'Pillow' }])
+    expect(string).toBe('Pillow')
+  })
+
+  it('verifies create string with product type and one option', () => {
```

Comment:
То же самое, насчет `verifies`. Ведь в тестах ниже этого слова нету)
```suggestion
  it('stringifies stack of product type and one option', () => {
```